src/pages/questionPage.js

```diff
@@ -24,6 +24,18 @@ export const initQuestionPage = () => {
   for (const [key, answerText] of Object.entries(currentQuestion.answers)) {
     const answerElement = createAnswerElement(key, answerText);
     answersListElement.appendChild(answerElement);
+
+    // making answers clickable
+    answerElement.addEventListener('click', function () {
```

Comment:
Is it possible to put this code block(ln28- ln38) into a separate function and just call it here? That why it we would reduce the complexity of **initQuestionPage** function.

Reply:
I didn't update this part in my last commit, sure I will add it